caiman/source_extraction/cnmf/params.py

```diff
@@ -6,13 +6,12 @@
 import pkg_resources
 from pprint import pformat
 import scipy
-from scipy.ndimage import generate_binary_structure, iterate_structure
+import scipy.ndimage
 
 import caiman.utils.utils
 import caiman.base.movies
-from ...paths import caiman_datadir
-from .utilities import dict_compare
-
+import caiman.paths
+import caiman.source_extraction.cnmf.utilities
```

Comment:
Not going to keep going through the remaining files, but I just want to say I really like the overall gist of what this PR does with making imports absolute. It makes things **way** more clean and clear! 🚀 

 Where I am quibbling is just in some of the details about where some of the specific cutoffs happen in the dots :smile: When reading the code, it can be a bit cumbersome/disruptive to wade through. Basically if there are things like `caiman.base.utils.visualize.cnmf.view.outlier` coming before a function, then we should trim it back. I would just focus on that but I think this is a really god improvement.